main.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"fmt"
+	"sort"
+)
+
+func printSorted(args map[int]string) {
+	keys := make([]int, 0, len(args))
+	for k := range args {
+		keys = append(keys, k)
+	}
+	sort.Ints(keys)
+	var sorted []string
+	for _, i := range keys {
+		sorted = append(sorted, args[i])
```

Comment:
It's better to print it right there and do not create separate variable to store it, but it's ok